shared/src/balancer/event_handler.rs

```diff
@@ -88,8 +88,8 @@ impl BalancerPoolRegistry {
     pub async fn get_pools_containing_token_pairs(
         &self,
         token_pairs: HashSet<TokenPair>,
-    ) -> Vec<RegisteredWeightedPool> {
-        let mut pool_set_1 = self
+    ) -> HashSet<H256> {
```

Comment:
nit: maybe this method should now be called get_pool_ids_containing_token_pairs?